app/update_stats.py

```diff
@@ -146,16 +161,20 @@ def collector(self):
                 _prev_time = _cur_time
                 with self._staging_changes() as staging_storage:
                     for st in self._staging:
-                        _suffix = st.op.value
-                        if _suffix in {"copy", "link", "download"}:
+                        if (
+                            isinstance(st.op, RegProcessOperation)
+                            and st.op != RegProcessOperation.OP_UNSPECIFIC
+                        ):
+                            _suffix = st.op.value
                             staging_storage.regular_files_processed += 1
+                            staging_storage.total_regular_file_size += st.size
```

Comment:
total_regular_file_size is static value. not dynamic value.

Reply:
Oh, I miss understanding this field.

Reply:
fixed in [d72a041](https://github.com/tier4/ota-client/pull/137/commits/d72a041b118aa22bcf01dcf6f50a80a321de2365), the real cause of the problem is that I set the total_regular_file_size before launching the update_stats_collecter.